.github/workflows/release.yml

```diff
@@ -40,29 +40,34 @@ jobs:
         with:
           python-version: '3.10'
 
+      # - name: Build wheels
+      #   uses: PyO3/maturin-action@v1
+      #   with:
+      #     target: ${{ matrix.target }}
+      #     container: ${{ matrix.docker_image }}
+      #     args: --release --out dist --find-interpreter --features dynamic
+      #     manylinux: auto
+      #     docker-options: |
+      #       -e RUST_BACKTRACE=1
+      #       -e DEP_JXL_LIB=${{ github.workspace }}/libjxl/build/lib
+      #       -e DEP_BROTLI_LIB=${{ github.workspace }}/libjxl/build/third_party/brotli
+      #       -e DEP_HWY_LIB=${{ github.workspace }}/libjxl/build/third_party/highway
+      #     before-script-linux: |
+      #       git clone --recurse-submodules --depth 1 -b v0.11.0 \
+      #         https://github.com/libjxl/libjxl.git
+      #       cd libjxl
+      #       cmake -B build -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF \
+      #         -DJPEGXL_ENABLE_TOOLS=OFF -DJPEGXL_ENABLE_DOXYGEN=OFF -DJPEGXL_ENABLE_MANPAGES=OFF \
+      #         -DJPEGXL_ENABLE_BENCHMARKS=OFF -DJPEGXL_ENABLE_EXAMPLES=OFF -DJPEGXL_ENABLE_JNI=OFF \
+      #         -DJPEGXL_ENABLE_SJPEG=OFF -DJPEGXL_ENABLE_OPENEXR=OFF
+      #       cmake --build build
+      #       cmake --install build
+      #       cd ..
+
       - name: Build wheels
```

Comment:
Hmmm, seems that the cargo cannot be configured with above commands: [CI_logs](https://github.com/Isotr0py/pillow-jpegxl-plugin/actions/runs/11722206612/job/32651250733?pr=79#step:5:462)... 

When we used maturin-action before, we don't need to care how to set Rust toolchain, but just to specify the target and let github-action set it automatically...

Reply:
True, it does shift the responsibility of Rust more onto you as the maintainer. In exchange, you have to deal with Python packaging shenanigans far less. Since the Rust ecosystem is entirely sane, I believe this is a worthwhile trade-off :)